src/Documentation/Alerts/readme.md

```diff
@@ -0,0 +1,2 @@
+Created the alerts with https://www.npmjs.com/package/sweetalert2-react
+You can see examples and instalation of the alerts inhttps://sweetalert2.github.io/
```

Comment:
please improve this documentation. First of all, add a title, a summary, a descriptions of the components and how to use them. Example:

## Alerts
### Summary

> what you are going describe here. What the purpose of this documenration?

### Components

> For each of them add case of use and description. If you can, a picture of how the components looks like

- Error: 
- Succes:
- Info:

### How to use it

```
someExampleCodeHere = () => {}
```

### Credits
Created the alerts with https://www.npmjs.com/package/sweetalert2-react
You can see examples and instalation of the alerts inhttps://sweetalert2.github.io/

Reply:
Also, add the link to this documentation on the main readme of the project.
> You can review documentation here of how to add styles to markdown: [Sintaxis de escritura y formato básicos
](https://docs.github.com/es/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
### Reusable components
- [Alerts](src/Documentation/Alerts/readme.md)

Reply:
Changes ready